app/components/add-contribution/component.js

```diff
@@ -21,13 +21,13 @@ export default Component.extend({
 
   init () {
     this._super(...arguments);
-    this.set('defaultDate', new Date());
+    this.set('defaultDate', new Date()); // TODO use beginning of current hour
```

Comment:
Someone has an idea for how to do this in the most simple and efficient way? /cc @galfert @fsmanuel

Reply:
I think it works like this:

```suggestion
    const defaultDate = new Date();
    defaultDate.setHours(0, 0, 0, 0);
    this.set('defaultDate', defaultDate);
```

Unfortunately you can't just concatenate it (`new Date().setHours(0, 0, 0, 0)`), because `setHours` returns a timestamp.

Reply:
Scrap my previous comment, I misread the beginning of the day, not the current hour.

Since we already have moment in the project, I think the easiest way would be

```suggestion
    this.set('defaultDate', moment().startOf('hour').toDate());
```

Reply:
Awesome, will try that. Thanks!